crates/turborepo-lockfiles/fixtures/pnpm-v9.yaml

```diff
@@ -0,0 +1,125 @@
+lockfileVersion: "9.0"
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
+importers:
+  .:
+    devDependencies:
+      turbo:
+        specifier: canary
+        version: 1.13.3-canary.1
+
+  apps/apps-a:
+    dependencies:
+      pkg-a:
+        specifier: workspace:*
+        version: link:../../packages/pkg-a
+      tooling-config:
+        specifier: workspace:*
+        version: link:../../packages/tooling-config
+
+  apps/apps-b:
+    dependencies:
+      tooling-config:
+        specifier: workspace:*
+        version: link:../../packages/tooling-config
+
+  packages/pkg-a:
+    dependencies:
+      tooling-config:
+        specifier: workspace:*
+        version: link:../tooling-config
+
+  packages/pkg-b:
+    dependencies:
+      tooling-config:
+        specifier: workspace:*
+        version: link:../tooling-config
+
+  packages/tooling-config: {}
+
+packages:
+  turbo-darwin-64@1.13.3-canary.1:
+    resolution:
+      {
+        integrity: sha512-1xfKCf/d7mD7oGT1qBnD5pYsZfI43Wftlo/N0GTpBLDjKhfJifBvng9ns5grhs79wNiIdDxYYGt6pHOUzc+6YQ==,
+      }
+    cpu: [x64]
+    os: [darwin]
+
+  turbo-darwin-arm64@1.13.3-canary.1:
+    resolution:
+      {
+        integrity: sha512-uNK9QnlDJBLuaR7l4/68AoKByQ+q+rT9OyvizzUV2KZ8r2u/Sv+f3GiV48qxtG1duYlWLlLnDGuEnUPKEq0WZg==,
+      }
+    cpu: [arm64]
+    os: [darwin]
+
+  turbo-linux-64@1.13.3-canary.1:
+    resolution:
+      {
+        integrity: sha512-D/RRtqHch0I5q/Pod7mgLPGrvjcDpvBe95yp5hNy59+ZSyKHlkDrm/pQaA/+6mdjHEQB8eIL/18nrKx/B5qduA==,
+      }
```

Comment:
umm.. what is this?? json inside yaml?

Reply:
They did in fact add JSON to YAML: https://www.yaml.info/learn/flowstyle.html

Reply:
look-how-theyve-massacred-my-boy.jpg